### PR TITLE
Update instructions for running tests locally.

### PR DESCRIPTION
**Pull request by rnewman**
This gets to the point of (failing) tests for me with a local Selenium grid.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integration/js/README.md [hunk 1]
```diff
@@ -2,13 +2,96 @@
...
+### Running the tests
+
+```shell
+export KITE_JAR="$KITE_HOME/KITE-Engine/target"
+cd integration
+export SELENIUM_GRID_PROVIDER=local
+```
+
+```shell
+$KITE_HOME/scripts/mac/path/r configs/meeting_end_test.config.json
```
Comment (ltrung) [inline integration/js/README.md:84]: Could we change meeting_end_test.config.json to something like <test_name>.config.json or say for example to run meeting end test:

File: integration/js/README.md [hunk 2]
```diff
@@ -2,13 +2,96 @@
...
 
+## Testing locally
+
+To run integration tests you will need:
+
+- Node 10 or higher. Node 14 is supported.
+- npm 6.11 or higher. 6.14.8 is supported.
+- [KITE](https://github.com/webrtc/KITE).
+
+### Installing KITE on macOS
```
Comment (ltrung) [inline integration/js/README.md:13]: This is common for both locally and SauceLabs right. Could we move it outside to before test locally?

Comment (ltrung) [reply integration/js/README.md:13]: Could we make it not specific to Mac OS? KITE has setup scripts for Linux and Windows too.

File: integration/js/README.md [hunk 3]
```diff
@@ -2,13 +2,96 @@
...
+```shell
+cd integration/js
+npm install
+cd -
+```
+
+### Running the tests
+
+```shell
+export KITE_JAR="$KITE_HOME/KITE-Engine/target"
```
Comment (ltrung) [inline integration/js/README.md:78]: We already did this step above.

File: integration/js/README.md (hunk 1 above)
Comment (ltrung) [reply integration/js/README.md:84]: We already add $KITE_HOME/scripts/mac/path/ to $PATH above so we could just do r configs/.. right?

File: integration/js/README.md (hunk 2 above)
Comment (rnewman) [reply integration/js/README.md:13]: We should, but I cannot write those docs.

File: integration/js/README.md (hunk 1 above)
Comment (rnewman) [reply integration/js/README.md]: No — `r` is a shell builtin, so even if it's in your path, it won't work.

**Review by ltrung — APPROVED** [review]

**Pull request merged**